Readme.md

```diff
@@ -35,7 +35,7 @@ python -m converter -c 30
 
 #### Unit tests
 For launching unit tests in root of the project:  
-```
+```bash & PowerShell
```

Comment:
@MariaMaximova, I think it would be better to split this code block into two with single language in each, because I cannot find any specification that allows several languages for one code block. Moreover, often the code for bash and PowerShell will be different (even if now they are the same).

Reply:
Done

Reply:
Was resolved. Please check [Readme.md](https://github.com/MariaMaximova/convert_temp/pull/19/files#diff-1550ec65ac92f65817fc28928dfef526912b5f52356ff43651369bae92f56031)